src/transformers/configuration_utils.py

```diff
@@ -94,6 +91,16 @@ def __init__(self, **kwargs):
         self.label2id = kwargs.pop("label2id", dict(zip(self.id2label.values(), self.id2label.keys())))
         self.label2id = dict((key, int(value)) for key, value in self.label2id.items())
 
+        # Tokenizer arguments TODO: eventually tokenizer and models should share the same config
+        self.prefix = kwargs.pop("prefix", "")
```

Comment:
should it be `None` instead of ""?

Reply:
I'd prefer `""` here because then we can save some if statements and just write: 

`[config.prefix + text for text in texts] `

in `pipelines.py`  for example.

Reply:
But then how are you going to not serialize it when calling `save_pretrained`?

We could do a getter for `self.prefix or ""` if it's easier syntax-wise

Reply:
Agree, will change that!

Reply:
My idea was to only serialize things that are different from the default config. 
So if `BartConfig.form_pretrained('bart-base-uncased')` is created, we compare it to `BartConfig()` before serializing and only save params that are different from `BartConfig()`, but it might be easier to just save all non `None` parameters.